google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/st/STBigQueryStorageLongRunningWriteTest.java

```diff
@@ -0,0 +1,211 @@
+
+  private static TableInfo tableInfo;
+  private static String tableId;
+  private static BigQuery bigquery;
+
+  private static JSONObject MakeJsonObject(int size) throws IOException {
+    JSONObject object = new JSONObject();
+    // size: (1, simple)(2,complex)()
+    if (size == 1) {
+      object.put("test_str", "aaa");
+      object.put("test_numerics", new JSONArray(new String[]{"1234", "-900000"}));
+      object.put("test_datetime", String.valueOf(LocalDateTime.now()));
+    }
+    else if (size == 2) {  // make it complicated and large
+      // TODO(jstocklass): Make a better json object that doesn't break the format rules.
+    }
+    return object;
+  }
+
+  @BeforeClass
+  public static void beforeClass() throws IOException {
+    //Assume.assumeTrue(LONG_TESTS_DISABLED_MESSAGE, Boolean.getBoolean(LONG_TESTS_ENABLED_PROPERTY));
+    client = BigQueryWriteClient.create();
+    parentProjectId = String.format("projects/%s", ServiceOptions.getDefaultProjectId());
+    RemoteBigQueryHelper bigqueryHelper = RemoteBigQueryHelper.create();
+    bigquery = bigqueryHelper.getOptions().getService();
+    DatasetInfo datasetInfo = DatasetInfo.newBuilder(/* datasetId = */ DATASET).setDescription(DESCRIPTION).build();
+    bigquery.create(datasetInfo);
+    LOG.info("Created test dataset: " + DATASET);
+    tableInfo =
+        TableInfo.newBuilder(
+            TableId.of(DATASET, TABLE),
+            StandardTableDefinition.of(
+                Schema.of(
+                    com.google.cloud.bigquery.Field.newBuilder("foo", LegacySQLTypeName.STRING)
+                        .setMode(Field.Mode.NULLABLE)
+                        .build())))
+            .build();
+    com.google.cloud.bigquery.Field.Builder innerTypeFieldBuilder =
+        com.google.cloud.bigquery.Field.newBuilder(
+            "inner_type",
+            LegacySQLTypeName.RECORD,
+            com.google.cloud.bigquery.Field.newBuilder("value", LegacySQLTypeName.STRING)
+                .setMode(Field.Mode.REPEATED)
+                .build());
+
+    bigquery.create(tableInfo);
+    tableId =
+        String.format(
+            "projects/%s/datasets/%s/tables/%s",
+            ServiceOptions.getDefaultProjectId(), DATASET, TABLE);
+    LOG.info(
+        String.format(
+            "%s tests running with parent project: %s",
+            ITBigQueryStorageLongRunningTest.class.getSimpleName(), parentProjectId));
+  }
+
+  @AfterClass
+  public static void afterClass() {
+    if (client != null) {
+      client.close();
+    }
+    if (bigquery != null) {
+      RemoteBigQueryHelper.forceDelete(bigquery, DATASET);
+      LOG.info("Deleted test dataset: " + DATASET);
+    }
+  }
+
+  @Test
+  public void testDefaultStream() throws IOException, InterruptedException, ExecutionException,
+      Descriptors.DescriptorValidationException {
+    // Set up a default stream. Write to it for a long time, (a few minutes for now) and make
+    // sure that everything goes well.
```

Comment:
This comment isn't accurate.  Should it have a TODO instead?

Reply:
This should be a TODO, will make change